src/main/java/emt/util/EMTConfigHandler.java

```diff
@@ -156,6 +160,7 @@ public static void syncConfig() {
         chanceOneRing = config.get(RANDOM, "One Ring spawning chance", 15, "If you have a lot of mods adding dungeon loot, you should definetely increase this").getInt();
         removeAmberAndCinnabarMacerating = config.get(RANDOM, "Clear Amber/Cinnabar Macerating", true, "This is here because of conflicts between mods that add the same input and different outputs").getBoolean(removeAmberAndCinnabarMacerating);
         impactOfRain = config.get(RANDOM, "Impact of rain", true, "The impact of rain on all wings").getBoolean(impactOfRain);
+        etherealMaceratorWhiteList.addAll(Arrays.asList(config.get(RANDOM, "White List for Ethereal Processor", new String[]{"ore", "cluster"}, "The allowed ore dictionary prefix for Ethereal Processor.").getStringList()));
```

Comment:
You said it's a prefix here, but you are trying to do a complete match by using `indexOf`. Just won't work.

Reply:
But indexOf check if the string starts from the same string, it doesn't check for equality.

Reply:
Ah well must have mistaken it into an indexOf into the list instead of String. Sorry for this. However the rest of points should still be valid

Reply:
`startsWith` would cause less confusion I guess
Also, it means stuff like "oreCrushed" is ok to duplicate?

Reply:
> Also, it means stuff like "oreCrushed" is ok to duplicate?

there is nothing has oredict "oreCrushed", only "crushed". if modder follows the name rule of oredict, there shouldn't be any problems.